### PR TITLE
Thomas/update cad scripts and examples

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
- add example for SimModeler workflow (would be added to the documentation in the next days)
- move example for Gocad workflow
- black on python scripts (does not hurt)

### CODE REVIEW

File: GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ 
+19.02.2016
```
Comment (sebwolf-de) [inline GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo]: Please update comment

File: GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Line(2) = {2, 3};
+Line(3) = {3, 4};
+Line(4) = {4, 1};
+Line Loop(5) = {1,2,3,4};
+Plane Surface(1) = {5};
+
+Extrude {0,0, Zmin} { Surface{1}; }
+
+Physical Surface(101) = {1,14,18,22,26,27};
+//Physical Surface(103) = {100,200};
```
Comment (sebwolf-de) [inline GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo]: Do we need these lines here?

**Review by sebwolf-de — COMMENTED** [review]
If I understand correctly, most changes are just replacing ' with " and adding some spaces, right? ~~Do you use some tools to check code style?~~

File: GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo (hunk 2 above)
Comment (Thomas-Ulrich) [reply GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo]: no

File: GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo (hunk 1 above)
Comment (Thomas-Ulrich) [reply GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo]: done

**Comment by Thomas-Ulrich** [comment]
> If I understand correctly, most changes are just replacing ' with " and adding some spaces, right? ~Do you use some tools to check code style?~

yes, (black) next time I do something similar, I will use a separate pull request, sorry about that.

**Comment by sebwolf-de** [comment]
No problem. I just wanted to make sure, that no logic has been changed there.

**Review by sebwolf-de — APPROVED** [review]

File: GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo (hunk 1 above)
Comment (krenzland) [reply GocadRelatedScripts/ExampleFiles/SimModeler_workflow/create_box.geo]: Additional comment (no changes required): In my opinion, you can also leave out the author information. I never change this when creating/updating files, it's included in the git information anyway.
But that's a matter of taste - do whatever you want :)

**Pull request merged**